src/components/PokeCard.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const PokeCard = ({ pokemon }) => {
+  return (
+    <>
+      <h1>{pokemon.name}</h1>
+      <img src={pokemon.sprites.front_default} alt="Normal Pokemon" />
+      <img src={pokemon.sprites.front_shiny} alt="Shiny Pokemon" />
+      <p>
+        <a href={pokemon.video} target="_blank" rel="noreferrer">
```

Comment:
Following what I said before, it makes the code cleaner here using less navigation through the pokemon obj.
```suggestion
        <a href={video} target="_blank" rel="noreferrer">
```
I know it is not much visible with this case but you'll see it for sure in more complex components.